src/store/embalm/actions.ts

```diff
@@ -145,15 +143,6 @@ export function toggleStep(step: Step): EmbalmActions {
   };
 }
 
-export function setDiggingFees(diggingFees: string): EmbalmActions {
-  return {
-    type: ActionType.SetDiggingFees,
-    payload: {
-      diggingFees,
-    },
-  };
-}
```

Comment:
Should the other code related to this get removed as well -- ActionType + related reducer code?